src/map.rs

```diff
@@ -2783,14 +2753,14 @@ where
                     let mut head = Shared::null();
                     let mut tail = Shared::null();
                     while !e.is_null() {
-                        // safety: either `e` is `bin`, in which case it is valid due to the above,
```

Comment:
why remove the backtics? I'd rather add more of them in than take them out if this is to make the comment more consistent.